packages/perseus/src/components/math-input.tsx

```diff
@@ -22,9 +21,14 @@ import _ from "underscore";
 
 import {debounce} from "../util/debounce";
 
+import {PerseusI18nContext} from "./i18n-context";
+
 import type {LegacyButtonSets} from "../perseus-types";
 import type {PerseusDependenciesV2} from "../types";
 import type {Keys, MathFieldInterface} from "@khanacademy/math-input";
+// NOTE(john): This is a public API and it's safe to import
+// eslint-disable-next-line monorepo/no-internal-import
+import type {MathInputStrings} from "@khanacademy/math-input/strings";
```

Comment:
Needed this for the type - I'm hoping that this will be tree-shaken on compilation.